fpt/src/lr35902.rs

```diff
@@ -710,21 +705,21 @@ impl LR35902 {
     }
 
     fn execute(&mut self, instruction: Instruction) {
-<<<<<<< HEAD
+        println!("{}", self.pc());
+        println!("{}", instruction);
+
+        if self.pc() == 0xFE {
+            std::process::exit(1);
+        }
```

Comment:
Why would the program counter jump to `0xFE`?

Reply:
Some debugging I was trying that made its way here